server/config/config.go

```diff
@@ -546,6 +569,61 @@ func (m *MDMConfig) AppleSCEP() (cert *tls.Certificate, pemCert, pemKey []byte,
 	return m.appleSCEP, m.appleSCEPPEMCert, m.appleSCEPPEMKey, nil
 }
 
+// AppleBM returns the parsed, validated and decrypted server token for Apple
+// Business Manager. It also parses and validates the Apple BM certificate and
+// private key in the process, in order to decrypt the token.
+func (m *MDMConfig) AppleBM() (tok *nanodep_client.OAuth1Tokens, err error) {
+	if m.appleBMToken == nil {
+		pair := x509KeyPairConfig{
+			m.AppleBMCert,
+			[]byte(m.AppleBMCertBytes),
+			m.AppleBMKey,
+			[]byte(m.AppleBMKeyBytes),
+		}
+		cert, err := pair.Parse(true)
+		if err != nil {
+			return nil, fmt.Errorf("Apple BM configuration: %w", err)
+		}
+		encToken, err := m.loadAppleBMEncryptedToken()
+		if err != nil {
+			return nil, fmt.Errorf("Apple BM configuration: %w", err)
+		}
+		bmKey, err := tokenpki.RSAKeyFromPEM(pair.keyBytes)
+		if err != nil {
+			return nil, fmt.Errorf("Apple BM configuration: parse private key: %w", err)
+		}
+		token, err := tokenpki.DecryptTokenJSON(encToken, cert.Leaf, bmKey)
+		if err != nil {
+			return nil, fmt.Errorf("Apple BM configuration: decrypt token: %w", err)
+		}
+		var jsonTok nanodep_client.OAuth1Tokens
+		if err := json.Unmarshal(token, &jsonTok); err != nil {
+			return nil, fmt.Errorf("Apple BM configuration: unmarshal JSON token: %w", err)
+		}
+		m.appleBMToken = &jsonTok
+	}
+	return m.appleBMToken, nil
```

Comment:
Are we checking if the token is expired here? See https://github.com/fleetdm/fleet/issues/8725#issuecomment-1332294940

Reply:
From the discussion, it seemed like we wanted all other validations to be done later/on first use, since the token can become expired soon after the start. I'm also wondering if doing non-Fleet-essential validations (i.e. not mysql/redis/etc.) at startup is the best approach vs having a better UI notification system, but that's a larger point I'll bring in Slack. For now, I think it's ok to add that check.